package_test.go

```diff
@@ -42,6 +61,7 @@ func TestPackage(t *testing.T) {
 			os.Setenv("PATH", Path)
 
 			WithoutArgs()
+			WithKeyPairValues()
```

Comment:
In hindsight, I realise there's no good reason to wrap these test cases in functions and might make it a bit hard to read the tests. 

Mind expanding `WithoutArgs` and `WithKeyPairValues` inline ?

Reply:
I'm not sure if I got it so, please, let me know if that's what you meant.

Reply:
@lucasvianna yup, this is as requested! thanks for making those changes